tests/association/test_positioning.py

```diff
@@ -306,3 +316,35 @@ def test_distance_between_points(input, expected):
 def test_manhatan_distance_between_points(input, expected):
     distance = Positioning.manhattan_distance_between_points(input[0], input[1])
     assert round(distance, 2) == expected
+
+def test_get_tokens_within_bounds(bbox_token_page):
+    box = generate_mapped_pred(300, 360, 290, 450, page_num=0)
+    positioning = Positioning()
+    bounds = positioning.get_tokens_within_bounds(box, bbox_token_page)
+    assert len(bounds) == 1
+    assert "true" in bounds[0]["text"]
+
+def test_get_tokens_within_bounds_excludes_overlap(bbox_token_page):
+    box = generate_mapped_pred(300, 360, 290, 450, page_num=0)
+    positioning = Positioning()
+    bounds = positioning.get_tokens_within_bounds(box, bbox_token_page)
```

Comment:
generally, best to explicitly write in the kwarg that you're testing even if it's the default